src/pyskindose/calculate_dose/calculate_irradiation_event_result.py

```diff
@@ -1,18 +1,19 @@
 from typing import List, Dict, Any
-
+from tqdm import tqdm
 import numpy as np
 import pandas as pd
-from pyskindose import Phantom, constants as const
+import logging
```

Comment:
This import should be moved to the top as it is part of Python standard library

Reply:
corrected